app/helpers/classes.js

```diff
@@ -0,0 +1,7 @@
+import { helper } from '@ember/component/helper';
+
+export function classes(classList) {
```

Comment:
https://github.com/JedWatson/classnames
can we use this? This way it will be the same as in react stack

Reply:
ha good one. For now I don't think we need such tool here
but I've renamed classes to classnames - I like that name better. and API of ot is very similar now